sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -145,6 +147,38 @@ public void testCopyFromIntoPartitionedTable() throws Exception {
     }
 
     @Test
+    public void testInsertIntoClosedPartition() throws Exception {
+        execute("create table t (n integer) partitioned by (n)");
+        ensureYellow();
+        execute("insert into t (n) values (1)");
+        refresh();
+
+        String[] indices = client().admin().indices().getIndex(new GetIndexRequest()).actionGet().getIndices();
+        client().admin().indices().close(new CloseIndexRequest(indices[0])).actionGet();
+
+        execute("select count(*) from t");
```

Comment:
if a partition was closed the count query would return 0, so i added test for that too